test/e2e/performance_test.go

```diff
@@ -0,0 +1,348 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+	"unsafe"
+
+	"golang.org/x/exp/rand"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	v1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	seed                            uint64 = 0xA1E47 // Use a specific rand seed to make the generated workloads always same
+	performanceAppLabel                    = "antrea-performance-test"
+	podsConnectionNetworkPolicyName        = "pods.ingress"
+	workloadNetworkPolicyName              = "workloads.ingress"
+	abImage                                = "antrea/apache-bench"
+	nginxImage                             = "nginx"
+	abContainerName                        = "apache-bench"
+	nginxContainerName                     = "nginx"
+)
+
+var (
+	benchNginxPodName = randPodName(abContainerName+"-")
+	benchABPodName    = randPodName(nginxContainerName+"-")
+
+	customizeTimes    = flag.Int("performance.http.times", 0, "Times of http requests")
+	customizeWorkload = flag.Int("performance.http.workload", 0, "Number of network policy workloads")
```

Comment:
same question as above, but for the customized network policy test

Reply:
Relied in prior comments.